R/logging-tools.R

```diff
@@ -88,87 +86,61 @@ log_button <- function(user_connection_data, input, button_id) {
 #' @rdname log_input
 #'
 #' @export
-log_custom_action <- function(user_connection_data, table_name = "user_log", values) {
-
-  if (!all(names(user_connection_data) == c("username", "session_id", "db_connection"))) {
-    stop("user_connection_data should be list of session_id and db_connection objects!")
-  }
-
-  if ("time" %in% names(values)) {
-    stop("You mustn't pass 'time' value into database. It is set automatically.")
-  }
-
-  send_query_df <- as.data.frame(
-    c(time = as.character(Sys.time()), values),
-    stringsAsFactors = FALSE
-  )
-
-  db <- user_connection_data$db_connection
-
-  odbc::dbWriteTable(
-    db,
-    table_name,
-    send_query_df,
-    overwrite = FALSE,
-    append = TRUE,
-    row.names = FALSE
-  )
-
+log_custom_action <- function(data_storage, table_name = "user_log", values) {
```

Comment:
We also have `log_session_detail()` that logs to `session_details`. 

I kept this as an argument solely because of this, but maybe we shouldn't allow unstructured writes to `session_details` bucket/table name